genesis/process/shardGenesisBlockCreator.go

```diff
@@ -40,6 +40,22 @@ type deployedScMetrics struct {
 	numOtherTypes int
 }
 
+func createGenesisConfig() config.GeneralSettingsConfig {
+	return config.GeneralSettingsConfig{
+		BuiltInFunctionsEnableEpoch:            0,
+		SCDeployEnableEpoch:                    unreachableEpoch,
```

Comment:
So, scDeployEnableEpoch was modified from 0 to unreachableEpoch ?

Reply:
Exactly. This was done as to not activate the gas computation changes that now depend on the same flag.